src/main/scala/org/constellation/crypto/KeyStoreUtils.scala

```diff
@@ -0,0 +1,41 @@
+package org.constellation.crypto
+
+import java.io.FileInputStream
+
+import cats.effect._
+import java.security.{Key, KeyPair, KeyStore, PrivateKey}
+
+object KeyStoreUtils {
+  // TODO: kpudlik: Consider using platform-agnostic PKCS12 because JKS works only with Java.
```

Comment:
@kpudlik Can you provide details in the new issue?

Reply:
Done https://github.com/Constellation-Labs/constellation/issues/639